quick-install/trow.yaml

```diff
@@ -101,6 +101,25 @@ spec:
           name: cert-vol
         - mountPath: /data
           name: data-vol
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8443
+            scheme: HTTPS
+            httpHeaders:
+            - name: HEALTH
+              value: "True"
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8443
+            scheme: HTTPS
+            httpHeaders:
+            - name: READINESS
+              value: "True"
+
+          initialDelaySeconds: 3
```

Comment:
How did you come up with the numbers? Unless there's a reason, I think we should leave it as the default.

Reply:
Will roll this back as well as the headers